sips/sip-029/sip-029-halving-alignment.md

```diff
@@ -0,0 +1,159 @@
+
+While ultimately halvings do need to happen, for those reasons, it would be highly preferable to not change the economic incentives until both Nakamoto and sBTC have been live for some time, as well as to have reductions in the STX block reward happen at the same time as reductions in the BTC block reward given their links.
+
+Therefore, this SIP proposes altering the token emission schedule to preserve the existing incentive structure while ensuring no increase in the 2050 target supply, and incorporating these principles:
+
+* All STX holders would see a reduced 2050 target STX supply by 0.19%, thereby making STX more scarce
+* All STX holders would see a reduced tail inflation after the final halving
+* Stacks miners and signers’ economic incentives would remain consistent with the existing incentives for the next 16 months
+* The new Stacks halving schedule would align with Bitcoin halvings starting in 2028 to strengthen the connection the Stacks L2 has to Bitcoin and also synchronize the economic adjustments of both assets, reducing changes in incentives for miners and signers at each halving, as further outlined in the Forum post [2]
+
+# Specification
+
+Applying these upgrades to the Stacks blockchain requires a consensus-breaking network upgrade, in this case, a hard fork. Like other such changes, this will require a new Stacks epoch. In this SIP, we will refer to this new epoch as Stacks 3.1.
+
+The _current_ STX emission schedule is presented as follows.  Note that the **first STX halving is in December 2024**.  The tail emission after the final halving in 2050 would be 125 STX per block, and the total supply at that time is projected to be 1,783,063,600 STX.
+
+| Coinbase Reward Reduction Phase | Approximate Date (time b/w halvings) | STX Reward (reduction) | STX Supply (after) |
+|--------------------------------|-------------------------------------|----------------------|-------------------|
+| Current                         | -                                   | 1000                 | -                  |
+| 1st                             | ~Dec 2024 (4 yrs)                  | 500 (50%)           | 1,512,993,315      |
+| 2nd                             | ~Dec 2028 (4 yrs)                  | 250 (50%)           | 1,645,084,888      |
+| 3rd                             | ~Dec 2032 (4 yrs)                  | 125 (50%)           | 1,689,389,675      |
+| -                               | ~Jan 2050 (17.08 yrs)             | 125 (0%)             | 1,783,063,600      |
+
+
+The _proposed_ STX emission schedule is presented as follows.  In particular, this SIP proposes preserving the 1000 STX coinbase until April 2026.  After this, there would be a reduction to 500 STX after two years, and another reduction to 125 STX after two more years.  The tail emission after the final reduction in 2036 would be 100 STX, and the total supply at that time is projected to be 1,783,063,600 STX (about 0.19% lower).
```

Comment:
```suggestion
The _proposed_ STX emission schedule is presented as follows.  In particular, this SIP proposes preserving the 1000 STX coinbase until April 2026.  After this, there would be a reduction to 500 STX after two years, and another reduction to 125 STX after two more years.  The tail emission after the final reduction in 2036 would be 100 STX, arriving at the total supply in January 2050 of 1,783,063,600 STX (about 0.19% lower than the current schedule).
```

I think this sentence needs this adjustment, then also fixing the numbers after confirming the correct values.

Reply:
Replaced this comment with an updated suggestion.